### DISCUSSION TITLE
Can't download the Resources file

### DISCUSSION THREAD

**Discussion by nkjcqvcpi** [Melledy/Grasscutter#2079] [Q&A]
I click the 'Resources' link from the wiki but asked to login. Then I register an account but told "Your account is pending approval from your GitLab administrator and hence blocked. Please contact your GitLab administrator if you think this is an error." I would like to know if it is a bug?

**Comment by NotThorny** [comment]
The wiki is outdated, use the current Yuuki repo: https://gitlab.com/YuukiPS/GC-Resources
